websites/demo-react-image-next/src/app/layout.tsx

```diff
@@ -0,0 +1,21 @@
+import "./global.css";
+import { Metadata } from "next";
+import { Providers } from "./providers";
+
+export const metadata: Metadata = {
+  title: "Suspensive - Visualization",
+};
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
```

Comment:
```suggestion
import { PropsWithChildren } from "react";

export const metadata: Metadata = {
  title: "Suspensive - Visualization",
};

export default function RootLayout({ children }: PropsWithChildren) {
```

`PropsWithChildren` is used elsewhere, so a review of the uniformity part.